packages/components/src/presentation-components/properties/UniquePropertyValuesSelector.tsx

```diff
@@ -166,7 +169,7 @@ function useUniquePropertyValuesLoader({ imodel, ruleset, fieldDescriptor }: Use
         keys: new KeySet(),
       });
       return {
-        options: content.items.filter((item) => item.displayValue !== undefined && item.displayValue !== ""),
+        options: content.items.filter((item) => item.displayValue !== undefined && item.displayValue !== nullStringIdentifier),
```

Comment:
I don't understand this - what should cause `getPagedDistinctValues` to return `nullStringIdentifier` as display value?

Reply:
Display value is set to `nullStringIdentifier` in [this line](https://github.com/iTwin/presentation/pull/278/files/2b15c69cc6dde5c85e34bd7873d58fa533370c62#diff-cd3479e259eb8fe117da9a547677eb07e3a6fdef707a4de33927511785bea27cR85) on _UniquePropertyValueSelector_ initial load. Thus, `getPagedDistinctValues` returns `content` with updated `displayValues`.

Reply:
Discussed in private